$MirOS: ports/misc/mc/patches/patch-configure_in,v 1.1 2005/03/18 15:50:27 tg Exp $
--- configure.in.orig	Fri Oct 15 17:50:18 2004
+++ configure.in	Fri Oct 15 17:52:57 2004
@@ -19,16 +19,6 @@ AC_ISC_POSIX
 
 AC_PROG_CC_STDC
 
-dnl
-dnl First try glib 2.x.  If it's not found, use glib 1.2.x.
-dnl Keep this check close to the beginning, so that the users
-dnl without any glib won't have their time wasted by other checks.
-dnl
-
-PKG_CHECK_MODULES(GLIB, [glib-2.0], , [glib_found=no])
-
-dnl Fall back to glib-1.2, don't use pkgconfig to find it.
-if test "x$glib_found" = "xno" ; then
 	dnl This temporary variable is a workaround for a bug in Autoconf-2.53
 	glib_path=$PATH:/usr/local/bin
 
@@ -38,7 +28,6 @@ if test "x$glib_found" = "xno" ; then
 	AC_ARG_VAR([GLIB_CONFIG], [Path to glib-config (version 1.2.x only)])
 	AM_PATH_GLIB(1.2.6,,[AC_MSG_ERROR([Test for glib failed.
 GNU Midnight Commander requires glib 1.2.6 or above.])])
-fi
 
 
 AC_HEADER_MAJOR
